activerecord/lib/active_record/railties/console_sandbox.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-ActiveRecord::Base.connection.begin_transaction(joinable: false)
-
-at_exit do
-  ActiveRecord::Base.connection.rollback_transaction
```

Comment:
No need to call `rollback_transaction` manually because the all connections will be rolled back on disconnected.

Reply:
Begin transaction